packages/expo-audio/src/ExpoAudio.ts

```diff
@@ -81,6 +81,20 @@ export function useAudioRecorderState(recorder: AudioRecorder, interval: number
   return state;
 }
 
+/**
+ * Creates an instance of an `AudioPlayer` that doesn't release automatically.
+ *
+ * > **info** For most use cases you should use the [`useAudioPlayer`](#useaudioplayersource-updateinterval) hook instead. See the [Using the AudioPlayer Directly](#using-the-audioplayer-directly) section for more details.
```

Comment:
```suggestion
 * > **info** For most use cases you should use the [`useAudioPlayer`](#useaudioplayersource-updateinterval) hook instead. See the [Using the `AudioPlayer` directly](#using-the-audioplayer-directly) section for more details.
```

Let's avoid capitalization of the word that is also not capitalize in the section's heading.